modules/cnf-topology-aware-lifecycle-operator-apply-policies.adoc

```diff
@@ -0,0 +1,107 @@
+// Module included in the following assemblies:
+// Epic CNF-2600 (CNF-2133) (4.10), Story TELCODOCS-285
+// * scalability_and_performance/cnf-topology-aware-lifecycle-operator.adoc
+
+:_content-type: PROCEDURE
+[id="talo-apply-policies_{context}"]
+= Applying update policies to managed clusters
+
+The Topology Aware Lifecycle Operator makes enforced copies of the managed {rh-rhacm} policies. Each copied policy has its own corresponding {rh-rhacm} placement rule and {rh-rhacm} placement binding. As the remediation plan progresses, the clusters are added one by one in the placement rule that corresponds with the applicable managed policy to the current cluster in the current batch. After a batch is updated, all clusters are removed from the placement rules associated with the copied policies. Then, the update of the next batch starts.
+
+.Prerequisites
+
+* Install the Topology Aware Lifecycle Operator.
+* Provision one or more managed SNO clusters using ZTP.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create the `ClusterGroupUpgrade` CR, and then save the YAML in the `cgu-cr.yaml` file:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1alpha1
+kind: ClusterGroupUpgrade
+metadata:
+  name: du-upgrade-4915
+  namespace: ztp-group-du-sno
+spec:
+  deleteObjectsOnCompletion: true
+  clusters: <1>
+  - cnfdb1
+  - cnfdb2
+  enable: true
+  managedPolicies:
+  - du-upgrade-platform-upgrade <2>
+  remediationStrategy:
+    maxConcurrency: 2 <3>
+    timeout: 240 <4>
+----
+<1> The list of clusters to update.
+<2> The name of the policy to apply.
+<3> The `maxConcurrency` field signifies the number of clusters updated at the same time.
+<4> The update timeout in minutes.
+
+. Create the `ClusterGroupUpgrade` CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f cgu-cr.yaml
+----
+
+.Verification
+
+. Check if the `ClusterGroupUpgrade` CR was created in the Hub cluster by running the following command:
++
+[source,terminal]
+----
+$ oc get cgu -A -w
```

Comment:
@alosadagrande I updated the name to correspond with the example I used.

@imiller0 If removing it doesn't change the output, I could delete it. Otherwise, I'd need a new output